src/special/webpack.js

```diff
@@ -53,10 +52,9 @@ function parseWebpack1(filepath, deps) {
 function mapRuleLoaders(module) {
   return module.rules
     .filter(rule => rule.loaders)
-    .map(rule => rule.loaders
-      .map(loader => ({
-        loader,
-      })));
+    .map(rule => rule.loaders.map(loader => ({
+      loader,
+    })));
```

Comment:
Sorry, I did change this but it looks like it's auto formatted back to this. I'll change it again.